### PR TITLE
[neeo] Made constants and handler packages internal

### PR DESCRIPTION
**Pull request by VelinYordanov**
See eclipse/smarthome#4330

Signed-off-by: VelinYordanov <velin.iordanov@gmail.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.neato/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -22,11 +22,7 @@ Import-Package:
  org.eclipse.smarthome.core.thing.type,
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.neato/META-INF/MANIFEST.MF]: I assume this file is not meant to be here.

File: addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/NeatoBindingConstants.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
  * which accompanies this distribution, and is available at
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/NeatoBindingConstants.java]: I assume this file is not meant to be here.

File: addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/NeatoHandlerFactory.java [hunk 3]
```diff
@@ -8,7 +8,7 @@
  */
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/NeatoHandlerFactory.java]: I assume this file is not meant to be here.

File: addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/discovery/NeatoAccountDiscoveryService.java [hunk 4]
```diff
@@ -18,10 +18,10 @@
 import org.eclipse.smarthome.config.discovery.DiscoveryResultBuilder;
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/discovery/NeatoAccountDiscoveryService.java]: I assume this file is not meant to be here.

File: addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/handler/NeatoAccountHandler.java [hunk 5]
```diff
@@ -6,7 +6,7 @@
  * which accompanies this distribution, and is available at
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/handler/NeatoAccountHandler.java]: I assume this file is not meant to be here.

File: addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/handler/NeatoHandler.java [hunk 6]
```diff
@@ -6,9 +6,9 @@
  * which accompanies this distribution, and is available at
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/handler/NeatoHandler.java]: I assume this file is not meant to be here.

**Comment by wborn** [comment]
I've rebased this PR so the Neato changes no longer show. These were already part of https://github.com/openhab/openhab2-addons/pull/3983 @cweitkamp.

**Pull request merged**